### PR TITLE
Add missing shortcut clickbox highlight to Trollweiss Mountain Cave

### LINKED ISSUE #9534: Missing clickbox from agility obstacle in Fremennik hunter area
There is no agility clickbox for this rock-slide in the fremennik hunter area, unlike all other obstacles.

1. Go to the northern part of the fremennik hunter area. (eagle is good way to get there)
2. Enter the cave to the north.
3. Rockslide with missing clickbox will be just beside you.

There should be a green box indicating where you can click to use the rockslide.

 - OS: Windows 8.1 & Xubuntu 16.04
 - RuneLite version: 1.5.29.1
 - Launcher version: 1.6.0

### PR DESCRIPTION
**Pull request by tortuga69**
Closes #9534

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/game/AgilityShortcut.java [hunk 1]
```diff
@@ -65,7 +65,9 @@
 			// Asgarnian Ice Dungeon
 			STEPS,
 			// Fossil Island Wyvern Cave
-			STAIRS_31485),
+			STAIRS_31485,
```
Comment (Darkkgreen) [inline runelite-client/src/main/java/net/runelite/client/game/AgilityShortcut.java]: Remove whitespace after ','

Comment (tortuga69) [reply runelite-client/src/main/java/net/runelite/client/game/AgilityShortcut.java]: I'm not sure if I understood correctly but I think that's what you wanted changed right?

Comment (Darkkgreen) [reply runelite-client/src/main/java/net/runelite/client/game/AgilityShortcut.java]: Yea, just to pass the tests

**Comment by dekvall** [comment]
Closes #9534

**Pull request merged**